linetools/isgm/abscomponent.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, radec, Zion, z, vlim, Ej=0./u.cm, A=None, Ntup=None, comment=
         self.name = ions.ion_name(self.Zion, nspace=1)
         self._abslines = []
 
-    def add_absline(self, absline, tol=0.1*u.arcsec):
+    def add_absline(self, absline, tol=0.1*u.arcsec, skip_vel=False):
```

Comment:
I prefer it the way it is.  
One has to purposefully choose skip_vel to 
use it.  And otherwise the abs_line is required
to fall within the Component.